### PR TITLE
Additional metrics

### PR DESCRIPTION
**Pull request by Elfoniok**
In this PR:

* Additional sentry handler for metric
* Use 1.2 Metric container which provides PSNR metric.

### CODE REVIEW

**Review by ktomala — APPROVED** [review]
LGTM

File: apps/blender/task/blendercropper.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def generate_split_data(self, resolution, image_border, splits_num,
         #  BoundBox2D border = cam->border.clamp();
         #  params.full_x = (int)(border.left * (float)width);
         #
-        #  NOTE BLENDER IS USING FLOATS Vgit stALUES
+        #  NOTE BLENDER IS USING FLOATS VALUES
```
Comment (Krigpl) [inline apps/blender/task/blendercropper.py:84]: image

File: apps/images.ini [hunk 2]
```diff
@@ -1,4 +1,4 @@
 golemfactory/base core/resources/images/Dockerfile 1.2 .
 golemfactory/blender blender/resources/images/Dockerfile 1.4 .
 golemfactory/luxrender lux/resources/images/Dockerfile 1.2 .
-golemfactory/image_metrics rendering/resources/images/Dockerfile 1.1 rendering/resources/images/
+golemfactory/image_metrics rendering/resources/images/Dockerfile 1.2 rendering/resources/images/
```
Comment (badb) [inline apps/images.ini:4]: What was changed in this docker image version? 
Do we want to update Dockerfile?

Comment (Elfoniok) [reply apps/images.ini:4]: Yes in 1.2 there is PSNR calculated also. Dockerfile was not changed. I think i will create separate repo for scripts for image_metrics or clean up exisitng ones.

**Comment by Elfoniok** [comment]
It was merged with b0.15.1

**Pull request closed without merging**